switchmap/poller/snmp/mib/generic/mib_lldp.py

```
@@ -95,32 +103,35 @@
+        )
 
-        # Get interface lldpRemSysCapEnabled data
-        values = self.lldpremsyscapenabled()
-        if values is not None:
-            for key, value in values.items():
-                final[key]["lldpRemSysCapEnabled"] = value
+        method_names = ["lldpRemSysName", "lldpRemSysDesc", "lldpRemPortDesc", "lldpRemSysCapEnabled"]
+        final = defaultdict(lambda: defaultdict(dict))
 
-        # Return
+        for i, (method_name, values) in enumerate(zip(method_names, results)):
```

Ruff (0.12.2)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)